eval/final/components.py

```diff
@@ -34,15 +34,97 @@ def run_final_eval_op(
     import os
 
     import torch
-    from helpers import (
-        VLLM_SERVER,
-        launch_vllm,
-        stop_vllm,
```

Comment:
I don't think these helpers are used anywhere else or are they? Would you like to delete them?

Reply:
They are not used anymore indeed. I guess I was hesitant to remove them, but let's do it.